autopush-common/Cargo.toml

```diff
@@ -76,8 +76,8 @@ actix-rt = "2.8"
 
 [features]
 # for testing:
-# default = ["emulator", "bigtable", "dynamodb"]
-default = ["dynamodb"]
+default = ["emulator", "bigtable", "dynamodb"]
```

Comment:
Do we want this change included? It's probably time to switch these on by default but this looks like a local edit (I would imagine switching it on by default would touch more `Cargo.toml`s?)

Reply:
well, we should probably remove `emulator` from the list.